suite-native/transactions/src/components/TransactionDetail/TransactionDetailParametersSheet.tsx

```diff
@@ -0,0 +1,103 @@
+    const formattedTransactionId = `#${transaction.txid.slice(0, 11)}...`;
+    const confirmationsCnt = getConfirmations(transaction, blockchain.blockHeight);
+
+    return (
+        <TransactionDetailSheet
+            isVisible={isVisible}
+            handleSheetVisibility={handleSheetVisibility}
+            title="Parameters"
+            iconName="warningCircle"
+            transactionId={transaction.txid}
+        >
+            <VStack marginBottom="medium">
+                <Card>
+                    <TransactionDetailRow title="Transaction ID">
+                        <Box flexDirection="row" alignItems="center" justifyContent="space-between">
+                            <Text variant="hint" color="gray1000">
+                                {formattedTransactionId}
+                            </Text>
+                            <Box marginLeft="medium">
+                                <IconButton
+                                    iconName="copy"
+                                    onPress={handleClickCopy}
+                                    isRounded
+                                    colorScheme="gray"
+                                    size="large"
+                                />
+                            </Box>
+                        </Box>
+                    </TransactionDetailRow>
+                    <TransactionDetailRow title="Confirmations">
+                        <Text color="gray1000">{confirmationsCnt} </Text>
+                        <Box marginLeft="small">
+                            <Icon name="confirmation" color="gray1000" />
+                        </Box>
+                    </TransactionDetailRow>
+                </Card>
+
+                <Card>
+                    {/* TODO: Uncoment when ethereum design is ready. */}
+                    {/* {transaction.symbol === 'eth' && (
+                    <>
+                        <TransactionDetailRow
+                            title="Gas limit"
+                            text={transaction.ethereumSpecific!.gasLimit}
+                        />
+                        <TransactionDetailRow
+                            title="Gas fee"
+                            text={transaction.ethereumSpecific!.gasUsed}
+                        />
+                        <TransactionDetailRow
+                            title="Nonce"
+                            text={transaction.ethereumSpecific!.nonce}
+                        />
+                    </>
+                )} */}
+
+                    <TransactionDetailRow
+                        title="Fee rate"
+                        text={`${getFeeRate(transaction)} ${getFeeUnits('bitcoin')}`}
+                    />
+                    {/* FIXME: There is no information about broadcast in transaction object. Where can i find it? */}
+                    {/* <TransactionDetailRow
+                    title="Broadcast"
+                    text={transaction ? 'Enabled' : 'Disabled'}
```

Comment:
I would remove this and just create an issue in the project

Reply:
I was hoping in solving this already in this PR. Maybe @Nodonisko can help with where to look for the broadcast transaction flag?

Reply:
I guess broadcast actually means that it have at least one confirmation? Please try to ask someone more into BTC like @mroz22 ?

Reply:
The result is that if a transaction has the `blockHeight` property, we can tell for sure it was broadcasted.